src/Opdex.Platform.WebApi/Controllers/DeployController.cs

```diff
@@ -83,13 +90,12 @@ public async Task<IActionResult> DeployDevModeEnvironment(LocalWalletCredentials
             await _mediator.Send(new ProcessGovernanceDeploymentTransactionCommand(governanceTokenTransaction.Hash), CancellationToken.None);
 
             // Deploy Market Deployer
-            var createDeployerRequest = new SmartContractCreateRequestDto(marketDeployerByteCode, request.WalletAddress, new string[0],
-                                                                          request.WalletName, request.WalletPassword);
+            var createDeployerRequest = new SmartContractCreateRequestDto(marketDeployerByteCode, request.WalletAddress, request.WalletName, request.WalletPassword);
 
             var deployerTransaction = await CallAndWait(
                 async () => await _mediator.Send(new CallCirrusCreateSmartContractCommand(createDeployerRequest), cancellationToken));
 
```

Comment:
Bet you thought "oh shit" when you went down the rabbit hole and saw you had to tweak this 😂 🤣

Reply:
Had to use Visual Studio for once 😆